azkaban-web-server/src/main/java/azkaban/webapp/servlet/ScheduleServlet.java

```diff
@@ -671,6 +676,11 @@ private void ajaxScheduleFlow(HttpServletRequest req,
     }
 
     List<SlaOption> slaOptions = null;
+    try {
+      slaOptions = getSlaOptionList(req, flowName);
+    } catch (Exception e) {
```

Comment:
I adapt the following coding style in ScheduleServlet
```
    ExecutionOptions flowOptions = null;
    try {
      flowOptions = HttpRequestUtils.parseFlowOptions(req);
      HttpRequestUtils.filterAdminOnlyFlowParams(userManager, flowOptions, user);
    } catch (Exception e) {
      ret.put("error", e.getMessage());
    }
```
Of course, it's possible to catch ServeletException, instead of Exception